nemo_skills/dataset/include/include_benchmark_utils.py

```
@@ -0,0 +1,313 @@
+        answer_prefix="Отговор: Нека мислим стъпка по стъпка.",
+        task='Следват въпроси с избираем отговор (с отговори) за {subject}. Мислете стъпка по стъпка и завършете отговора си с "{ans_suffix}", където X е буквата на правилния избор.',
+        placeholder="отговорът е ({})",
+    ),
+    "Croatian": MCQFormat(
+        answer_prefix="Odgovor: Razmislimo korak po korak.",
+        task='Slijede pitanja s višestrukim izborom (s odgovorima) o {subject}. Razmišljajte korak po korak i završite svoj odgovor s "{ans_suffix}", gdje je X slovo točnog odgovora.',
+        placeholder="odgovor je ({})",
+    ),
+    "Dutch": MCQFormat(
+        answer_prefix="Antwoord: Laten we stap voor stap nadenken.",
+        task='Hieronder staan ​​meerkeuzevragen (met antwoorden) over {subject}. Denk stap voor stap en eindig uw antwoord met "{ans_suffix}", waarbij X de letter van het juiste antwoord is.',
```

Ruff (0.15.15)

[error] Invalid unescaped character zero-width-space, use "\u200B" instead

Replace with escape sequence

(PLE2515)